cidata.py

```diff
@@ -561,74 +568,86 @@ def read(
         enabled = podnet_b
         disabled = podnet_a
     elif podnet_a_enabled is True and podnet_b_enabled is True:
-        return False, data_dict, messages[3216]
+        retval = False
+        message_list.append(messages[3216])
     elif podnet_a_enabled is False and podnet_b_enabled is False:
-        return False, data_dict, messages[3217]
+        retval = False
+        message_list.append(messages[3217])
     else:
-        return False, data_dict, messages[3218]
+        retval = False
+        message_list.append(messages[3218])
 
```

Comment:
If any "message" has been raised so far, the remainder of this read function will also fail. 

I think if there is an issue with any of the values in config.json we should return here and only continue if the config.json is valid.

Reply:
Makes sense, yeah.